align/compiler/compiler.py

```diff
@@ -132,7 +132,7 @@ def compiler_output(
             continue
         gen_const = [True for const in subckt.constraints if isinstance(const, constraint.Generator)]
         if not gen_const:
-            with open(result_dir/ f"{subckt.name}.const.json", "w") as f:
+            with open(result_dir / f"{subckt.name.lower()}.const.json", "w") as f:
```

Comment:
@kkunal1408 , reverting to lower case so that user can simply copy the file.

Reply:
Both lower and upper case file  names would work.